whitespace_checks.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# Check trailing whitespaces
+
+trailing_whitespaces=$(grep -I '[[:blank:]]$' -Rl . | grep -v -E '.*(spago|.git|dist-newstyle|node_modules|.node|generated-docs|dist|test-data|.spago2nix).*')
```

Comment:
We should omit looking into the directories in the first command, instead of letting it grep everything and then filter.

Also, please add `tmp` to the list of excluded dirs, it's used in E2E